typedload/dataloader.py

```diff
@@ -222,8 +223,9 @@ def __init__(self, **kwargs) -> None:
             (is_literal, _literalload),
             (is_typeddict, _typeddictload),
             (lambda type_: type_ in {datetime.date, datetime.time, datetime.datetime}, _datetimeload),
-            (lambda type_: type_ == datetime.timedelta, _timedeltaload),
+            (is_pattern, _patternload),
             (lambda type_: type_ in self.strconstructed, _strconstructload),
+            (lambda type_: type_ == datetime.timedelta, _timedeltaload),
```

Comment:
why did you move this line?

Reply:
I think it was a remnant of earlier debugging but it should not be needed, undid this, thanks for pointing it out.